app/Main.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
Since you're only using this for testing, I don't think we should be committing this file.

Reply:
As we discussed, it should instead be moved to an `examples` folder in a project of its own.

Reply:
done